mmv1/templates/terraform/examples/secure_source_manager_branch_rule_basic.tf.erb

```diff
@@ -0,0 +1,17 @@
+resource "google_secure_source_manager_instance" "instance" {
+    location = "us-central1"
+    instance_id = "my-instance"
```

Comment:
Can you use a variable for this similar to how repository_id is populated?

This will cause issues with tests when they run in parallel and try to create the same instance

Reply:
Done.